apps/cli/src/node/default.env

```diff
@@ -1,76 +1,26 @@
 # s6-overlay
-S6_STAGE2_HOOK="/etc/s6-overlay/scripts/stage2-hook.sh"
-S6_VERBOSITY="${S6_VERBOSITY:-2}"
+S6_VERBOSITY="2"
 S6_CMD_WAIT_FOR_SERVICES_MAXTIME=${SM_DEADLINE_MACHINE:-30000}
-# global
-RUST_LOG="${RUST_LOG:-info}"
-## shared
-### redis
-REDIS_ENDPOINT=${REDIS_ENDPOINT:-redis://127.0.0.1:6379}
-### contract-address-file
-DAPP_CONTRACT_ADDRESS_FILE=/usr/share/sunodo/dapp.json
-### session-id
-SESSION_ID=default_session_id
-### chain-id
-CHAIN_ID=31337
-TX_CHAIN_ID=31337
-### server-manager-endpoint
-SERVER_MANAGER_ENDPOINT=http://localhost:5001
-SERVER_MANAGER_ADDRESS=localhost:5001
-SERVER_MANAGER_LOG_LEVEL="${SERVER_MANAGER_LOG_LEVEL:-info}"
-REMOTE_CARTESI_MACHINE_LOG_LEVEL="${REMOTE_CARTESI_MACHINE_LOG_LEVEL:-info}"
-### state-server-endpoint
-SC_GRPC_ENDPOINT=http://localhost:50051
 
-### database
-POSTGRES_ENDPOINT="${POSTGRES_ENDPOINT:-postgres://postgres:password@database:5432/postgres}"
-
-# dispatcher
-## uses redis
-## uses chain-id
-## uses state-server-endpoint
-DAPP_DEPLOYMENT_FILE=/usr/share/sunodo/dapp.json
-ROLLUPS_DEPLOYMENT_FILE=/usr/share/sunodo/localhost.json
-RD_EPOCH_DURATION="${RD_EPOCH_DURATION:-86400}"
-SC_DEFAULT_CONFIRMATIONS=1
-
-# authority-claimer
-## uses redis
-## uses state-server-endpoint
-## uses chain-id (TX_CHAIN_ID acctually)
-TX_CHAIN_IS_LEGACY: "false"
-TX_DEFAULT_CONFIRMATIONS: "2"
-TX_SIGNING_MNEMONIC="test test test test test test test test test test test junk"
-TX_PROVIDER_HTTP_ENDPOINT=http://anvil:8545
-
-# state-server
-SF_GENESIS_BLOCK=1
-SF_SAFETY_MARGIN=1
-BH_HTTP_ENDPOINT=http://anvil:8545
-BH_WS_ENDPOINT=ws://anvil:8545
-BH_BLOCK_TIMEOUT="${BLOCK_TIMEOUT:-8}"
-
-# advance-runner
-## uses redis
-## uses contract-address-file
-## uses session-id
-## uses chain-id
-## uses server-manager-endpoint
-PROVIDER_HTTP_ENDPOINT=http://anvil:8545
-SNAPSHOT_DIR="/var/opt/cartesi/machine-snapshots"
-SNAPSHOT_LATEST="/var/opt/cartesi/machine-snapshots/latest"
-
-# inspect-server
-## uses session-id
-## uses server-manager-endpoint (acctually uses SERVER_MANAGER_ADDRESS)
-INSPECT_SERVER_ADDRESS=0.0.0.0:5005
-
-# indexer
-## uses redis
-## uses contract-address-file
-## uses chain-id
-
-# graphql-server
-## uses postgres-config
-GRAPHQL_HOST=0.0.0.0
-GRAPHQL_PORT=4000
\ No newline at end of file
+# cartesi/rollups-node
+CARTESI_LOG_LEVEL="info"
+CARTESI_LOG_TIMESTAMP="true"
+CARTESI_HTTP_PORT="10000"
+CARTESI_FEATURE_HOST_MODE="false"
+CARTESI_FEATURE_READER_MODE="false"
+CARTESI_EPOCH_DURATION="120"
```

Comment:
default was 86400

Reply:
I think 120 is a good default for devnet. I used https://github.com/cartesi/rollups-node/blob/main/build/compose-devnet.yaml as referece.

Reply:
I think 120 is too short.

We want to allow the developer to understand that there is a point in time in the application life that the vouchers he created don't have proofs yet, and will have proofs eventually, after the epoch closes.

I think having a 120 value (2 minutes) increases the chances of him not noticing that, because proofs are created too "immediately".

I think a better value is 1 hour.